style.css

```diff
@@ -679,7 +721,7 @@ a {
   }
 
```

Comment:
## partner logo

```

.partner-logo {
  opacity: 0.5;
}
```

- Hey, let's kindly add the `filter: grayscale(100%) contrast(30%);` to the partner images to make it have the gray color to make the color more like that on the main design to improve our design.